### DISCUSSION TITLE
Word-level timestamps in new version of Whisper

### DISCUSSION THREAD

**Discussion by dgoryeo** [linto-ai/whisper-timestamped#50] [Q&A] [answered]
I was just wondering if the implmentation of word-level timestamps in the latest version of Whisper would open more opportunities/capabilities/improvements to whisper-timetsamped? Just curious if that has any impact.

**Comment by Jeronymous** [comment, marked as answer]
Very relevant point. It has impact for sure.
I am starting to benchmark the word-level timestamps provided by the new version of Whisper (on a dataset for which there is some groundtruth for the word timestamps).
And also whisper-timetsamped needs some updates to follow new modifications in Whisper (working on it in priority).
